src/script/lua_api/l_minimap.cpp

```diff
@@ -140,8 +138,11 @@ int LuaMinimap::l_show(lua_State *L)
 	LuaMinimap *ref = checkobject(L, 1);
 	Minimap *m = getobject(ref);
 
-	if (m->getMinimapMode() == MINIMAP_MODE_OFF)
-		m->setMinimapMode(MINIMAP_MODE_SURFACEx1);
+	// This is not very adapted to new minimap mode management. Btw, tried
```

Comment:
it's perfectly fine to break CSM compatibility if needed - we don't officially support the API, so can break it any time

Reply:
Ok, I left this as is, it could be reworked later.